newscoop/application/configs/parameters/parameters.yml

```diff
@@ -134,3 +134,4 @@ parameters:
         environment: 'prod'
     simple_security:
         users_file: "%application_path%/configs/security/users.json"
+    remember_me.lifetime: 31536000 # 1 year in seconds
```

Comment:
we can't use settings from system preferences and set it this same way as we set in backend?